src/lib-components/Navbar/index.tsx

```diff
@@ -216,6 +215,7 @@ export const Navbar = ({
                 <HamburgerButton
                   isOpen={sideMenuIsOpen}
                   onClick={handleToggleSideMenu}
+                  links={sideMenuLinks}
```

Comment:
The button shouldn't be aware of links, rather it should not exist if there are no links